commit b4c58fedcd3771521f6fdab31c448ffea9419d5b (refs/changes/56/756/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-21T09:59:40-07:00 (3 years ago)
    
    TOOLS-1583 typo in 'sdcadm platform set-default' handler
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/lib/platform.js b/lib/platform.js
index a4298e7..74342ee 100644
--- a/lib/platform.js
+++ b/lib/platform.js
@@ -2280,7 +2280,7 @@ function do_set_default(subcmd, opts, args, cb) {
             ' option is not given'));
     }
 
-    if (opts.plaform === 'help') {
+    if (opts.platform === 'help') {
         cb(new errors.UsageError(
         'Please use `sdcadm platform help set-default` instead'));
     }
